src/sinks/console-json-sink.cpp

```diff
@@ -36,6 +37,13 @@ void ConsoleJSONSink::dump(Log const& log,
             nlohmann::json log_json(construct_log_json(log, channel, context_info, global_context_info));
             log_json["host"] = host_;
             log_json["service"] = service_;
+            if (log_thread_id_)
+            {
+                std::ostringstream oss;
+                oss << std::this_thread::get_id();
+                log_json["context_info"]["thread_id"] = oss.str();
```

Comment:
Consider using the following:
```suggestion
                log_json["context_info"]["thread_id"] = fmt::to_string(std::this_thread::get_id());
```
- It is shorted and `to_string` makes it fairly obvious what it is supposed to do
- Using libfmt here is supposedly slightly more performant

Reply:
![Screenshot 2025-06-19 at 9 01 49](https://github.com/user-attachments/assets/87b5cc89-1b21-4698-8ec0-2936e1ae3f28)